server/src/main/java/org/apache/kafka/server/ReplicaFetch.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.server;
+
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.requests.FetchRequest;
+
+import java.util.Map;
+
+public class ReplicaFetch {
+    private final Map<TopicPartition, FetchRequest.PartitionData> partitionData;
+    private final FetchRequest.Builder fetchRequest;
+
+    public ReplicaFetch(Map<TopicPartition, FetchRequest.PartitionData> partitionData,
+                        FetchRequest.Builder fetchRequest) {
+        this.partitionData = partitionData;
+        this.fetchRequest = fetchRequest;
+    }
+
+    public Map<TopicPartition, FetchRequest.PartitionData> getPartitionData() {
+        return partitionData;
+    }
+
+    public FetchRequest.Builder getFetchRequest() {
+        return fetchRequest;
+    }
```

Comment:
Kafka typically avoids using get as a method name prefix, thus these method should be `partitionData` and `fetchRequest`.